SEED-platform/seed
# Feat/init inventory list beta

#### Any background context you want to provide?
- SEED is creating a paginated view of inventory

#### What's this PR do?
- create a new frontend page for paginating inventory data
  - benchmark with 10,000 properties (local, docker-compose setup)
    - original table: ~30 seconds to finish loading -- loads all properties
    - new table: ~10 seconds to finish loading first page (~2 secs on subsequent pages) -- loads 100 properties per page

#### How should this be manually tested?
- go to the new "beta" inventory list page (should be a tab you can click)


!!! Note to the reviewer(s): The first commit is a _direct copy_ of the original files, so don't worry about reviewing those changes!

#### What are the relevant tickets?
#3039 

#### Screenshots (if appropriate)
<img width="1179" alt="Screen Shot 2021-12-20 at 10 15 43 AM" src="https://user-images.githubusercontent.com/18518728/146790016-c7b12d4b-c78b-4b63-a131-b3172ac6a320.png">


https://user-images.githubusercontent.com/18518728/146789841-1c1844bd-f938-4c5d-9dac-67a380cfdb01.mov
